graphql/schemabuilder/pagination.go

```diff
@@ -17,6 +17,54 @@ type Connection struct {
 	PageInfo   PageInfo
 }
 
+// paginateManually applies the pagination arguments to the edges in memory and sets hasNextPage +
+// hasPrevPage. The behavior is expected to conform to the Relay Cursor spec:
+// https://facebook.github.io/relay/graphql/connections.htm#EdgesToReturn()
+func (c *Connection) paginateManually(args PaginationArgs) error {
```

Comment:
what does manually mean here? the name made me think it's the case we assume the field func paginates bt we still call `applyCursorsToAllEdges`

Reply:
Yeah, that's fair. Maybe paginate internally?